src/services/utilities.ts

```diff
@@ -1335,18 +1335,30 @@ namespace ts {
         let position = 0;
         // However we should still skip a pinned comment at the top
         if (ranges.length && ranges[0].kind === SyntaxKind.MultiLineCommentTrivia && isPinnedComment(text, ranges[0])) {
-            position = ranges[0].end + 1;
+            position = ranges[0].end;
+            AdvancePastLineBreak();
             ranges = ranges.slice(1);
         }
         // As well as any triple slash references
         for (const range of ranges) {
             if (range.kind === SyntaxKind.SingleLineCommentTrivia && isRecognizedTripleSlashComment(node.text, range.pos, range.end)) {
-                position = range.end + 1;
+                position = range.end;
+                AdvancePastLineBreak();
                 continue;
             }
             break;
         }
         return position;
+
+        function AdvancePastLineBreak() {

```

Comment:
would `skipTrivia` be sufficient here?

Reply:
Wouldn't that skip other whitespace and comments?

Reply:
Though I am confused as to why this is upper camel case instead of lower camel case like every other helper function .

Reply:
That would be because I'm a C# dev. :wink:

Reply:
but you are at the end of the line anyways.. so comments and white spaces are not an issue here..

Reply:
Good point.  I notice, though, that `skipTrivia` doesn't handle the other linebreak characters.  Unless you object, I'll leave it like this for the sake of explicitness.